scripts/gis-material-update/process/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.10
+
+WORKDIR /opt
+COPY ./process/requirements.txt /opt
+RUN python -m venv /opt/venv && /opt/venv/bin/pip install --upgrade pip && /opt/venv/bin/pip install -r requirements.txt
```

Comment:
This step fails on my machine:
```
❯ docker-compose build
[+] Building 11.3s (16/17)
 => [haitaton-gis_gis-fetch internal] load build definition from Dockerfile                                                                                                                                     0.0s
 => => transferring dockerfile: 276B                                                                                                                                                                            0.0s
 => [haitaton-gis_gis-process internal] load build definition from Dockerfile                                                                                                                                   0.0s
 => => transferring dockerfile: 307B                                                                                                                                                                            0.0s
 => [haitaton-gis_gis-fetch internal] load .dockerignore                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                 0.0s
 => [haitaton-gis_gis-process internal] load .dockerignore                                                                                                                                                      0.0s
 => => transferring context: 2B                                                                                                                                                                                 0.0s
 => [haitaton-gis_gis-fetch internal] load metadata for docker.io/library/ubuntu:latest                                                                                                                         0.7s
 => [haitaton-gis_gis-process internal] load metadata for docker.io/library/python:3.10                                                                                                                         1.2s
 => [haitaton-gis_gis-fetch 1/4] FROM docker.io/library/ubuntu@sha256:20fa2d7bb4de7723f542be5923b06c4d704370f0390e4ae9e1c833c8785644c1                                                                          0.0s
 => CACHED [haitaton-gis_gis-fetch 2/4] RUN apt-get update &&     apt-get install -y     software-properties-common wget gdal-bin                                                                               0.0s
 => CACHED [haitaton-gis_gis-fetch 3/4] RUN add-apt-repository ppa:rmescandon/yq                                                                                                                                0.0s
 => CACHED [haitaton-gis_gis-fetch 4/4] RUN apt install yq                                                                                                                                                      0.0s
 => [haitaton-gis_gis-fetch] exporting to image                                                                                                                                                                 0.0s
 => => exporting layers                                                                                                                                                                                         0.0s
 => => writing image sha256:4aee9e0683340b9de326c844d2c0571a3e2aa1217c8ea0c6d21e77093d592fe9                                                                                                                    0.0s
 => => naming to docker.io/library/haitaton-gis_gis-fetch                                                                                                                                                       0.0s
 => [haitaton-gis_gis-process 1/5] FROM docker.io/library/python:3.10@sha256:e9c35537103a2801a30b15a77d4a56b35532c964489b125ec1ff24f3d5b53409                                                                   0.0s
 => [haitaton-gis_gis-process internal] load build context                                                                                                                                                      0.0s
 => => transferring context: 72B                                                                                                                                                                                0.0s
 => CACHED [haitaton-gis_gis-process 2/5] WORKDIR /opt                                                                                                                                                          0.0s
 => CACHED [haitaton-gis_gis-process 3/5] COPY ./process/requirements.txt /opt                                                                                                                                  0.0s
 => ERROR [haitaton-gis_gis-process 4/5] RUN python3 -m venv /opt/venv && /opt/venv/bin/pip3 install --upgrade pip && /opt/venv/bin/pip3 install -r requirements.txt                                            9.9s
------
 > [haitaton-gis_gis-process 4/5] RUN python3 -m venv /opt/venv && /opt/venv/bin/pip3 install --upgrade pip && /opt/venv/bin/pip3 install -r requirements.txt:
#0 2.435 Requirement already satisfied: pip in ./venv/lib/python3.10/site-packages (22.2.2)
#0 2.890 Collecting pandas
#0 3.019   Downloading pandas-1.5.0-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (11.4 MB)
#0 3.953      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.4/11.4 MB 12.1 MB/s eta 0:00:00
#0 3.989 Collecting gtfs_kit
#0 3.997   Downloading gtfs_kit-5.2.3-py3-none-any.whl (63 kB)
#0 4.005      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 63.2/63.2 kB 19.3 MB/s eta 0:00:00
#0 4.036 Collecting pyyaml
#0 4.047   Downloading PyYAML-6.0-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (733 kB)
#0 4.123      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 733.7/733.7 kB 9.8 MB/s eta 0:00:00
#0 4.264 Collecting numpy>=1.21.0
#0 4.276   Downloading numpy-1.23.3-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (13.9 MB)
#0 5.443      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.9/13.9 MB 11.6 MB/s eta 0:00:00
#0 5.521 Collecting pytz>=2020.1
#0 5.528   Downloading pytz-2022.2.1-py2.py3-none-any.whl (500 kB)
#0 5.572      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 500.6/500.6 kB 11.8 MB/s eta 0:00:00
#0 5.601 Collecting python-dateutil>=2.8.1
#0 5.610   Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
#0 5.629      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 14.3 MB/s eta 0:00:00
#0 5.650 Collecting json2html>=1
#0 5.658   Downloading json2html-1.3.0.tar.gz (7.0 kB)
#0 5.663   Preparing metadata (setup.py): started
#0 5.784   Preparing metadata (setup.py): finished with status 'done'
#0 5.901 Collecting folium>=0
#0 5.920   Downloading folium-0.12.1.post1-py2.py3-none-any.whl (95 kB)
#0 5.929      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 95.0/95.0 kB 12.8 MB/s eta 0:00:00
#0 6.046 Collecting utm>=0
#0 6.054   Downloading utm-0.7.0.tar.gz (8.7 kB)
#0 6.059   Preparing metadata (setup.py): started
#0 6.154   Preparing metadata (setup.py): finished with status 'done'
#0 6.171 Collecting geopandas>=0
#0 6.189   Downloading geopandas-0.11.1-py3-none-any.whl (1.0 MB)
#0 6.266      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 13.8 MB/s eta 0:00:00
#0 6.356 Collecting pycountry>=19
#0 6.365   Downloading pycountry-22.3.5.tar.gz (10.1 MB)
#0 7.179      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.1/10.1 MB 12.5 MB/s eta 0:00:00
#0 7.599   Installing build dependencies: started
#0 8.535   Installing build dependencies: finished with status 'done'
#0 8.536   Getting requirements to build wheel: started
#0 8.677   Getting requirements to build wheel: finished with status 'done'
#0 8.678   Preparing metadata (pyproject.toml): started
#0 8.816   Preparing metadata (pyproject.toml): finished with status 'done'
#0 8.869 Collecting shapely>=1
#0 8.881   Downloading Shapely-1.8.4-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (1.3 MB)
#0 8.981      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.3/1.3 MB 12.7 MB/s eta 0:00:00
#0 9.011 Collecting requests>=2
#0 9.021   Downloading requests-2.28.1-py3-none-any.whl (62 kB)
#0 9.027      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.8/62.8 kB 12.4 MB/s eta 0:00:00
#0 9.148 Collecting rtree>=0
#0 9.159   Downloading Rtree-1.0.0-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (938 kB)
#0 9.234      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 938.2/938.2 kB 13.1 MB/s eta 0:00:00
#0 9.357 Collecting branca>=0.3.0
#0 9.365   Downloading branca-0.5.0-py3-none-any.whl (24 kB)
#0 9.390 Collecting jinja2>=2.9
#0 9.397   Downloading Jinja2-3.1.2-py3-none-any.whl (133 kB)
#0 9.410      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.1/133.1 kB 11.6 MB/s eta 0:00:00
#0 9.434 Collecting packaging
#0 9.440   Downloading packaging-21.3-py3-none-any.whl (40 kB)
#0 9.447      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.8/40.8 kB 6.3 MB/s eta 0:00:00
#0 9.487 Collecting fiona>=1.8
#0 9.504   Downloading Fiona-1.8.21.tar.gz (1.0 MB)
#0 9.586      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 12.9 MB/s eta 0:00:00
#0 9.646   Preparing metadata (setup.py): started
#0 9.729   Preparing metadata (setup.py): finished with status 'error'
#0 9.732   error: subprocess-exited-with-error
#0 9.732
#0 9.732   × python setup.py egg_info did not run successfully.
#0 9.732   │ exit code: 1
#0 9.732   ╰─> [2 lines of output]
#0 9.732       Failed to get options via gdal-config: [Errno 2] No such file or directory: 'gdal-config'
#0 9.732       A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable.
#0 9.732       [end of output]
#0 9.732
#0 9.732   note: This error originates from a subprocess, and is likely not a problem with pip.
#0 9.733 error: metadata-generation-failed
#0 9.733
#0 9.733 × Encountered error while generating package metadata.
#0 9.733 ╰─> See above for output.
#0 9.733
#0 9.733 note: This is an issue with the package mentioned above, not pip.
#0 9.733 hint: See above for details.
------
failed to solve: executor failed running [/bin/sh -c python3 -m venv /opt/venv && /opt/venv/bin/pip3 install --upgrade pip && /opt/venv/bin/pip3 install -r requirements.txt]: exit code: 1
```

Reply:
I'm running a silicon-architecture mac, which might have something to do with this.

Reply:
Adding an explicit install of GDAL seems to solve this.
```
RUN apt-get update
RUN apt-get install -y gdal-bin libgdal-dev
```

Reply:
Would be nice to understand this a bit better (why this happens).